Makefile

```diff
@@ -25,12 +27,26 @@ dist/calico_kubernetes: kubernetesbuild.created
 	-e PYTHONPATH=/code/calico_kubernetes \
 	calico/kubernetes-build pyinstaller calico_kubernetes/calico_kubernetes.py -a -F -s --clean
 
-ut: dist/calico_kubernetes
+ut: binary
 	docker run --rm -v `pwd`/calico_kubernetes:/code/calico_kubernetes \
 	-v `pwd`/nose.cfg:/code/nose.cfg \
 	calico/kubernetes-build bash -c \
-	'/tmp/etcd -data-dir=/tmp/default.etcd/ >/dev/null 2>&1 & \
-	PYTHONPATH=/code/calico_kubernetes nosetests calico_kubernetes/tests -c nose.cfg'
+	'>/dev/null 2>&1 & PYTHONPATH=/code/calico_kubernetes \
+	nosetests calico_kubernetes/tests -c nose.cfg'
+
+# UT runs on Cicle
+ut-circle: binary
```

Comment:
Fine to make the ut-circle rely on it though, so that we can assert the binary gets built when Circle is checking stuff.